module-services/service-cellular/call/api/CallTimer.hpp

```diff
@@ -26,7 +25,7 @@ namespace call::api
 class CallTimer : public call::api::Timer
 {
     sys::TimerHandle handle;
-    std::time_t startActiveTime{};
+    std::time_t startActiveTime = 0;
```

Comment:
I'd keep brace initialization `{0}` instead of assignment.

Reply:
Fixed!